src/allencell_ml_segmenter/core/event.py

```diff
@@ -61,6 +64,11 @@ class Event(Enum):
     ACTION_CURATION_SEG1_THREAD_ERROR = "curation_seg1_thread_error"
     ACTION_CURATION_RAW_THREAD_ERROR = "curation_raw_thread_error"
 
+    # Thresholding events
+    ACTION_EXECUTE_THRESHOLDING = "execute_thresholding"
+    ACTION_SAVE_THRESHOLDING_IMAGES = "save_thresholding_images"
+    ACTION_THRESHOLDING_DISABLED = "disable_thresholding"
```

Comment:
As discussed in a ticket, i think it would be better if our ux design didnt require `ACTION_THRESHOLDING_DISABLED`